src/components/ArchivedReportFooter.js

```diff
@@ -22,11 +22,11 @@ const propTypes = {
             /** The reason the report was closed */
             reason: PropTypes.string.isRequired,
 
-            /** (For accountMerged reason only), the email of the previous owner of this report. */
-            oldLogin: PropTypes.string,
+            /** (For accountMerged reason only), the accountID of the previous owner of this report. */
+            oldAccountID: PropTypes.number,
 
-            /** (For accountMerged reason only), the email of the account the previous owner was merged into */
-            newLogin: PropTypes.string,
+            /** (For accountMerged reason only), the accountID of the account the previous owner was merged into */
```

Comment:
```suggestion
            /** (For accountMerged reason only), the accountID of the previous owner was merged into */
```

the accountID of the **previous** owner was merged into? seems like a typo,

Reply:
Oh nvm I missed this comment https://github.com/Expensify/App/pull/21844#issuecomment-1613465096